README.md

```diff
@@ -8,6 +8,14 @@ Manage [RabbitMQ](https://www.rabbitmq.com/) clusters deployed to [Kubernetes](h
 
 The operator deploys RabbitMQ `3.8.5`, and requires a Kubernetes cluster of `1.16` or above.
 
+## Versioning
+
+RabbitMQ Cluster Kubernetes Operator follows non-strict [semver](https://semver.org/). More details about other versioning strategies that were considered can be found
+[here](https://github.com/rabbitmq/cluster-operator/blob/main/docs/proposals/20200806-versioning-release-strategy.md). 
```

Comment:
I would delete the second sentence since it's already mentioned in version_guidelines.md and is not that relevant for the main README.